Desktop/analysis/analysis.cpp

```diff
@@ -394,7 +390,6 @@ Json::Value Analysis::asJSON(bool withRSource) const
 	analysisAsJson["rfile"]			= _rfile;
 	analysisAsJson["hasReport"]		= _hasReport;
 	analysisAsJson["progress"]		= _progress;
-	analysisAsJson["version"]		= _version.asString();
```

Comment:
Its still useful to know which version of JASP was used to make an analysis right?